### PR TITLE
Update feature roadmap

### PR DESCRIPTION
**Pull request by anawishnoff**
Updating roadmap doc based on new changes we announced at the community call.

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -105,49 +107,50 @@ See the image below for a more detailed look at our plans for getting WinUI 3.0
 ## WinUI 3.0 Feature Roadmap
 We know that for many of our developers, the choice to adopt WinUI 3 is dependent on certain features and capabilities. The table below outlines the status on WinUI 3 features, and the expected date they'll be available to you. 
 
+As we begin switching towards releasing preview and stable packages in parallel, you'll see in the feature roadmap below that some features become "preview" features (marked with a *) at a certain release. This means that the feature is not stable/bug-free enough to be included in the stable release, but will be available in the corresponding pre-release package.
```
Comment (ryandemopoulos) [inline docs/roadmap.md:110]: I think the part about preview features having an asterisk is no longer true...?  did you mean a diamond?

Comment (anawishnoff) [reply docs/roadmap.md]: Thanks Ryan, missed this. Just addressed in my last commit

**Review by ryandemopoulos — APPROVED** [review]
Looks good; ready to merge IMO.

**Pull request merged**